validator/validator.go

```diff
@@ -0,0 +1,63 @@
+package validator
```

Comment:
In go, the package name is part of the "verb" when calling a function or type. So usually, when naming a package and its members, try to think of how they would look like when being used.

I would rename this package to `validate` and remove the `Validate` prefix from all functions, since it is clear from the package what they do.
This way, when calling a function in the package it would be: `validate.Annotations(...)`.

For example, the go stdlib does this all the time: `io.Reader()`, `bytes.Buffer`, etc...

Reply:
renamed to validate